app/controllers/review_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ReviewController < ApplicationController
+
+  def index
+    @reviews = Review.all
+  end
```

Comment:
I don't see an index page for reviews anywhere in your code further down! If you aren't building out a page for those, don't have this action/route